### PR TITLE
fix:resolve another Razzle peerDependencies that depends on old versi…

### PR DESCRIPTION
**Pull request by pancaspe87**
`"webpack-dev-server": "^3.11.0"` is another Razzle peer dependencies. Unfortunately it relies on  `node-forge v0.10.0` which is  a vulnerability.
This PR will reesolve the peer dependency to a higher version

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,8 @@
     "mini-css-extract-plugin": "^1.6.2",
     "nth-check": "^2.0.1",
     "react": "^17.0.2",
-    "typescript": "^4.5.5"
+    "typescript": "^4.5.5",
+    "webpack-dev-server": "^4.7.4"
```
Comment (JakeLane) [inline package.json:65]: should this be added in the ssr package?

Comment (pancaspe87) [reply package.json:65]: I thought you need to add it in both. It is in the SSR package

Comment (JakeLane) [reply package.json:65]: In theory, packages should only be in the root package.json if the dependency is not for a specific package and is required to run the repo.

Comment (pancaspe87) [reply package.json:65]: It seems that we'll still download the old version if I don't place it in the root package.json

**Review by JakeLane — APPROVED** [review]

**Pull request merged**